src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,74 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  total: number
+  perPage: number
+  currentPage: number
+  onPageChange:(number: number) => void
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesLength = total / +perPage;
+
+  const createPageList = () => {
```

Comment:
Instead of using this code as a function and adding function calls to JSX, you could refactor it to a constant and directly use it in JSX without the need for function calls.

Reply:
you are very responsible for your work